lib/modulesync.rb

```diff
@@ -84,10 +84,16 @@ def self.manage_file(filename, settings, options)
       Renderer.remove(module_file(options[:project_root], module_name, filename))
     else
       templatename = local_file(options[:configs], filename)
+      template_file = if !File.exist?("#{templatename}.erb") && File.exist?(templatename)
```

Comment:
It's a bit ugly that we have to duplicate this logic from the renderer. I wonder if we should rip that out and do a major version bump.

Reply:
I agree it's ugly, I added it only after I saw failing tests :)

I have no strong opinion on ripping it out, but I do think the whole ".erb extension is optional" and doing weird things like this to work around that in the code is rather confusing.

Reply:
I think it's sort of halfway in the migration. The original had no `.erb` extensions. Then it became the normal way and 'naked' filenames were deprecated. The migration was never completed, but the idea was to copy 'naked' filenames without templating them. IMHO we complete it and do a major version bump.